Jenkinsfile

```diff
@@ -76,18 +76,35 @@ def cleanup_master() {
 	}
 }
 
+def archive_logs() {
+	sh '''
+	# need a unique and descriptive name so that the artifacts don't clobber each other
+	# and we can know which log file goes to which node, etc
+	# this also removes a / that would otherwise be included
```

Comment:
Comment is outdated (the part about removing a forward slash) and might not be needed at all any more.